Source/Services/RPSLS.Web/wwwroot/css/main.css

```diff
@@ -130,6 +134,17 @@ a.secondary-link {
     transition: background-color 0.2s ease-in-out, transform 0.1s ease-in-out;
 }
 
+.custom-button[disabled] .custom-button-link {
+    background-color: #d9007d;
+    text-shadow: none;
+    cursor: auto;
+    opacity: 0.5;
+}
+
+.custom-button[disabled]:active .custom-button-link {
+    transform: none;
+}
+
```

Comment:
Add disabled as class as has no meaning as attribute.
fyi: it does not affect accessibility.

Reply:
Changes done, thanks!